client/state/ui/selectors.js

```diff
@@ -46,6 +46,16 @@ export function getSectionName( state ) {
 }
 
 /**
+ * Returns the current section group name.
+ *
+ * @param  {Object}  state Global state tree
+ * @return {?String}       Current section group name
+ */
+export function getGroupName( state ) {
```

Comment:
Would `getSectionGroup` be a more suitable name, given the other selector `getSectionName` ? (And more accurately reflecting the path property it's retrieving)